### ISSUE TITLE
Write a user guide with tips for tuning

### ISSUE THREAD

**Issue by psiul** [psiul/ICDE2019-GPU-Join#1]

**Comment by louchenyao** [comment]
@psiul I am interested in this excellent work and want to evaluate them on V100 GPU. Is there any near plan to add a README on how to build and run this project? Thanks!

**Comment by Eilowangfang** [comment]
> _No description provided._

Yes. I'm also interested in your ICDE'19 paper, looking forward to a detailed README or instructions.
